reminder/module.py

```diff
@@ -445,6 +445,11 @@ async def reminder_reschedule(self, ctx, idx: int, datetime_str: str):
         print_date = utils.time.format_datetime(date)
 
         embed = await self._get_embed(ctx, query)
+        embed.add_field(
+            name=_(ctx, "Original time"),
```

Comment:
You have to run the popie, otherwise it won't pass the build as this won't be translated.